pkg/relayer/config/types.go

```diff
@@ -96,11 +86,11 @@ type RelayMinerPocketNodeConfig struct {
 type RelayMinerProxyConfig struct {
 	// ProxyName is the name of the proxy server, used to identify it in the config
 	ProxyName string
-	// Type is the transport protocol used by the proxy server like (http, https, etc.)
-	Type ProxyType
-	// Host is the host on which the proxy server will listen for incoming
+	// ServerType is the transport protocol used by the proxy server like (http, https, etc.)
+	ServerType ServerType
+	// ListenAddress is the host on which the proxy server will listen for incoming
```

Comment:
Should this be ListenURL?

Reply:
Not in the context of this struct. The `YAMLRelayMinerSupplierConfig` which is the exact representation of the config file has `ListenUrl` and embeds the listening address and the server type in its `<server_type>://<listen_address>`.

While `RelayMinerProxyConfig` get the parsed version, with the extracted `ServerType` and `ListenAddress` components to be able to start a server and listen to the provided address.

We are effectively changing the config file structure but not the `RelayMiner` logic that uses it. This is where having the both structs came handy! Thank to your's and Bryan's reviews back then :)